packages/terra-application/src/notification-dialog/NotificationDialog.jsx

```diff
@@ -2,14 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
I recommend viewing this entire file. Git has treated this as change+move of the temporary dev-site file, but the temporary files weren't ever reviewed.